app/models/period.rb

```diff
@@ -1,6 +1,6 @@
 class Period
   REGISTRATION_BUFFER_MONTHS = 3
-  REPORTING_TIMEZONE = CountryConfig.current[:time_zone] || "Asia/Kolkata"
+  REPORTING_TIME_ZONE = CountryConfig.current[:time_zone] || "Asia/Kolkata"
```

Comment:
Unrelated, but maybe defaulting to Asia/Kolkata isn't valid anymore?

Reply:
That would make sense - I think the alternative is to require this be set in the country config, else we raise an error and blow up on application boot.